src/test/java/gov/healthit/chpl/aqa/stepDefinitions/DisclosureUrlSteps.java

```diff
@@ -50,7 +50,7 @@ public void iAmOnListingDetailsPageOfListingWithCHPLID(final String chplId) {
      */
     @Then("^it should show updated url$")
     public void itShouldShowUpdatedUrl() {
-        String actualString = SearchPage.disclosureUrl(driver).getText();
+        String actualString = ListingDetailsPage.disclosureUrl(driver).getText();
```

Comment:
Should pull in from upstream. This test has changed to have the actual text of the disclosure URL

Reply:
I think updated test is not yet on chpladmin/chpl-test, maybe the changes aren't merged yet? When I pull from upstream, everything shows up-to-date.